pullrequest.rst

```diff
@@ -105,23 +105,23 @@ Here is a quick overview of you can contribute to CPython on GitHub:
 
 1.  If an Issue doesn't exist, `create an Issue`_ that describes your change
 
-2.  Prepare your system by :ref:`setup`
+2.  :ref:`Get started <setup>`_ and set up your system
```

Comment:
Cross references don't use `_` prefix, so you should remove these; this is why travis failed on you. In general, run `make html` on your changes to catch all warnings :-). Apart from that, LGTM :-)